src/fixtures/translation_plugin_template/README.md

```diff
@@ -0,0 +1,23 @@
+A Kibana translation plugin structure. 
```

Comment:
I love the intention here, but I'm not a fan of having an unused template as a part of the source. Fixtures are for tests, so if we wanted to make use of them that way I would feel better about it, but this would be best as a pull-request to [generator-kibana-plugin](https://github.com/elastic/generator-kibana-plugin)